### PR TITLE
feat: Added Navigation

### PR DESCRIPTION
**Pull request by krishanu2911**
Added navigation component with different links and title
Screenshot (425)

### CODE REVIEW

**Review by sadathshariff — COMMENTED** [review]
Everything looks good

File: CSS/components/navigation.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 1rem;
+}
+.nav-right{
+    display: flex;
+    gap: 1rem;
+    margin-left: auto;
+}
```
Comment (shuntzhere) [inline CSS/components/navigation.css]: Hey, can you let me know what this margin left auto is doing, as I think auto works when there is fixed width and I don't see it in your styles. Let me know.

**Review by shuntzhere — COMMENTED** [review]
Looks good Krishanu, let's discuss the margin-left auto, I am not sure how is that doing anything.

File: CSS/components/navigation.css (hunk 1 above)
Comment (thesudhansupanda) [reply CSS/components/navigation.css:19]: The auto keyword will give the left side a share of the remaining space.

Comment (krishanu2911) [reply CSS/components/navigation.css]: ya margin auto is setting both icon which is inside the nav-right to the right corner as vikram suggestion to use margin-left auto in navbar to set the icon to right

**Review by shuntzhere — APPROVED** [review]
Looks good to merge.

**Pull request merged**